lib/bumblebee/text/m2m100.ex

```diff
@@ -0,0 +1,516 @@
+defmodule Bumblebee.Text.M2m100 do
+  alias Bumblebee.Shared
+
+  options =
+    [
+      vocab_size: [
+        default: 128_112,
+        doc: """
+        the vocabulary size of the token embedding. This corresponds to the number of distinct
+        tokens that can be represented in model input and output
+        """
+      ],
+      max_positions: [
+        default: 1024,
+        doc: """
+        the vocabulary size of the position embedding. This corresponds to the maximum sequence
+        length that this model can process. Typically this is set to a large value just in case,
+        such as 512, 1024 or 2048
+        """
+      ],
+      hidden_size: [
+        default: 1024,
+        doc: "the dimensionality of hidden layers"
+      ],
+      encoder_num_blocks: [
+        default: 12,
+        doc: "the number of Transformer blocks in the encoder"
+      ],
+      decoder_num_blocks: [
+        default: 12,
+        doc: "the number of Transformer blocks in the decoder"
+      ],
+      encoder_num_attention_heads: [
+        default: 16,
+        doc: "the number of attention heads for each attention layer in the encoder"
+      ],
+      decoder_num_attention_heads: [
+        default: 16,
+        doc: "the number of attention heads for each attention layer in the decoder"
+      ],
+      encoder_intermediate_size: [
+        default: 4096,
+        doc:
+          "the dimensionality of the intermediate layer in the transformer feed-forward network (FFN) in the encoder"
+      ],
+      decoder_intermediate_size: [
+        default: 4096,
+        doc:
+          "the dimensionality of the intermediate layer in the transformer feed-forward network (FFN) in the decoder"
+      ],
+      scale_embedding: [
+        default: true,
+        doc: "scale embeddings by dividing by sqrt(hidden_size)"
+      ],
+      activation: [
+        default: :relu,
+        doc: "the activation function"
+      ],
+      dropout_rate: [
+        default: 0.1,
+        doc: "the dropout rate for encoder and decoder"
+      ],
+      attention_dropout_rate: [
+        default: 0.1,
+        doc: "the dropout rate for attention weights"
+      ],
+      activation_dropout_rate: [
+        default: 0.0,
+        doc: "the dropout rate for activations inside fully connected layers"
+      ],
+      initializer_scale: [
+        default: 0.02,
+        doc:
+          "the standard deviation of the normal initializer used for initializing kernel parameters"
+      ]
+    ] ++
+      Shared.common_options([:num_labels, :id_to_label]) ++
+      Shared.token_options(pad_token_id: 1, eos_token_id: 2, decoder_start_token_id: 2)
```

Comment:
```suggestion
    ] ++ Shared.token_options(pad_token_id: 1, eos_token_id: 2, decoder_start_token_id: 2)
```

(we don't have classification architecture here, so these are not relevant)